hoover_script.py

```diff
@@ -60,17 +218,21 @@ def run(target):
             ])
 
         tmp = Path(_tmp)
-        download(VIRTUALENV_URL, tmp)
-        download(SETUPTOOLS_URL, tmp)
-        download(PIP_URL, tmp)
+        download(VIRTUALENV_URL.get(), tmp)
+        download(SETUPTOOLS_URL.get(), tmp)
+        download(PIP_URL.get(), tmp)
         yield run
 
 def git_clone(url, directory):
     runcmd(['git', 'clone', url], cwd=str(directory))
 
-def preflight():
+def migrate():
     manage_py('search', 'migrate')
     manage_py('snoop', 'migrate')
+
+def preflight(skip_migrations=False):
+    if not skip_migrations:
```

Comment:
`if not skip_migrations` is also a double-negation, how about `if run_migrations`?